pages/_app.js

```diff
@@ -1,10 +1,19 @@
 import GlobalStyle from "../components/GlobalStyle";
+import { useState } from "react";
 
 function MyApp({ Component, pageProps }) {
+  const [flowAsanaIds, setFlowAsanaIds] = useState([]);
+  const [selectedAsanaIds, setSelectedAsanaIds] = useState([]);
   return (
     <>
       <GlobalStyle />
-      <Component {...pageProps} />
+      <Component
```

Comment:
Hey Cathrin! off course you can do it like this. But maybe you would like to consider wrapping your  <Component {...pageProps} around a Layout Component and put your StateUpLiftings in there. Would look more cleaner I think

Reply:
But not required

Reply:
Thanks for the hint. I will add a Layout Component in a later user story (when I add navigation etc). I will try to put the states in there then.